java/src/apps/AppsBundle.properties

```diff
@@ -22,6 +22,8 @@ MenuItemPaste = Paste
 MenuItemPreferences = Preferences...
 MenuItemPreferencesProfile = Profiles...
 
+MenuView = View
```

Comment:
These two new entries should be in jmri.NamedBeanBundle instead of here (The contents of NamedBeanBundle are available to *all* Bundle classes, and items in the jmri... packages should not be relying on anything in the apps... packages (I know, some do, but we are trying to avoid that)).